### ISSUE TITLE
MinMax

### ISSUE THREAD

**Issue by mgbriones** [IIC2433/Syllabus-2020-2#115]
Hola tengo una duda de como funciona esta normalizacion. 
Si estoy tratando de obtener una normalizacion para un vector (1,8) ,que representa una fila de mi dataframe,  en la formula el min_a seria 1 , max_a seria 8? 

PD: cuando intente comprobar mis resultados con MinMaxScaler().fit_transform() me dan resultados distintos.

![image](https://user-images.githubusercontent.com/53794443/101177020-142bbd00-3626-11eb-89d1-422ce80ffc67.png)

**Comment by AstridESMJ** [comment]
¡Hola!

La normalización min-max:
                                                                    _zi = xi − min(x) / max(x)−min(x)_

con _x=(x1,...,xn)_ y _zi_ es el _iésimo_ dato normalizado.

Saludos,

Astrid
